### PR TITLE
fix(modaldialog): addresses long titles; focus-visible close state

### PR DESCRIPTION
**Pull request by dzucconi**
Addressing some minor issues that arose when Q/Aing the new auth dialog.

### CODE REVIEW

File: packages/palette/src/elements/Text/Text.tsx [hunk 1]
```diff
@@ -65,9 +65,17 @@ export const overflowEllipsisMixin = css`
...
+export const hyphenate = css`
+  word-wrap: break-word;
+  word-break: break-word;
+  hyphens: auto;
+`
+
 /** TextProps */
 export type TextProps = BaseTextProps &
   BoxProps & {
+    hyphenate?: boolean
```
Comment (dzucconi) [inline packages/palette/src/elements/Text/Text.tsx:78]: New `Text` feature.

File: packages/palette/src/elements/ModalDialog/ModalDialogContent.tsx [hunk 2]
```diff
@@ -104,3 +109,13 @@ export const ModalDialogContent: React.FC<ModalDialogContentProps> = ({
     </Flex>
   )
 }
+
+const Close = styled(Clickable)`
+  transition: opacity 250ms;
```
Comment (damassi) [inline packages/palette/src/elements/ModalDialog/ModalDialogContent.tsx:114]: Don't we have this stored somewhere as a CONST?

Comment (dzucconi) [reply packages/palette/src/elements/ModalDialog/ModalDialogContent.tsx:114]: We don't. I don't really think we need to, unless there was some agreed upon values and guidelines for animations and transitions in the DS Figma (there probably should be) — I'm just basing values off of personal experience with what feels good. In this case it's just a focus so, actually, we should maybe not have any focus just so it's a bit more apparent.

**Comment by artsyit** [comment]
:rocket: PR was released in [`@artsy/palette-charts@26.1.0`](https://github.com/artsy/palette/releases/tag/%40artsy/palette-charts%4026.1.0), [`@artsy/palette@27.1.0`](https://github.com/artsy/palette/releases/tag/%40artsy/palette%4027.1.0) :rocket:

**Pull request merged**